pages/career/applyjob/[jobID].page.js

```diff
@@ -0,0 +1,474 @@
+              </div>
+            </div>
+
+            <div className="tw-font-sans tw-w-full tw-sm:w-1/2">
+              <label
+                htmlFor="currentSalary"
+                className="tw-font-sans tw-block tw-text-gray-700 tw-mb-1"
+              >
+                Current Salary(LPA)
+              </label>
+              <input
+                type="text"
+                id="currentSalary"
+                name="currentSalary"
+                value={formData.currentSalary}
+                onChange={handleChange}
+                className="tw-font-sans tw-w-full tw-px-3 tw-py-2 tw-border tw-border-gray-300 tw-rounded-lg tw-bg-gray-100 tw-text-gray-800"
+                placeholder="2"
+              />
+            </div>
+          </div>
+
+          <div className="tw-font-sans tw-mb-4 tw-flex tw-flex-wrap tw-gap-4">
+            <div className="tw-font-sans tw-w-full tw-sm:w-1/2">
+              <label
+                htmlFor="expectedSalary"
+                className="tw-font-sans tw-block tw-text-gray-700 tw-mb-1"
+              >
+                Expected Salary(LPA)
+              </label>
+              <input
+                type="text"
+                id="expectedSalary"
+                name="expectedSalary"
+                value={formData.expectedSalary}
+                onChange={handleChange}
+                className="tw-font-sans tw-w-full tw-px-3 tw-py-2 tw-border tw-border-gray-300 tw-rounded-lg tw-bg-gray-100 tw-text-gray-800"
+                placeholder="3"
+              />
+            </div>
+
+            <div className="tw-font-sans tw-w-full tw-sm:w-1/2">
+              <label
+                htmlFor="availableToJoin"
+                className="tw-font-sans tw-block tw-text-gray-700 tw-mb-1"
+              >
+                Available To Join (in days)
+              </label>
+              <input
+                type="number"
+                id="availableToJoin"
+                name="availableToJoin"
+                value={formData.availableToJoin}
+                onChange={handleChange}
+                className="tw-font-sans tw-w-full tw-px-3 tw-py-2 tw-border tw-border-gray-300 tw-rounded-lg tw-bg-gray-100 tw-text-gray-800"
+                placeholder="7"
+              />
+            </div>
+          </div>
+
+          <div className="tw-font-sans tw-mb-4">
+            <label
+              htmlFor="currentLocation"
+              className="tw-font-sans tw-block tw-text-gray-700 tw-mb-1"
+            >
+              Current Location
+            </label>
+            <input
+              type="text"
+              id="currentLocation"
+              name="currentLocation"
+              value={formData.currentLocation}
+              onChange={handleChange}
+              className="tw-font-sans tw-w-full tw-px-3 tw-py-2 tw-border tw-border-gray-300 tw-rounded-lg tw-bg-gray-100 tw-text-gray-800"
+              placeholder="Aligarh"
+            />
+          </div>
+
```

Comment:
we have library for dropdown showing all locations. So user can easily select it's location